assets/src/edit-story/components/form/switch.js

```diff
@@ -120,9 +121,16 @@ function Switch({ value, disabled, onChange, onLabel, offLabel }) {
     },
     [onChange]
   );
+  const ref = useRef();
+  useKeyDownEffect(
+    ref,
+    ['space', 'enter', 'left', 'right'],
```

Comment:
Is it expected that if the value is already on the "right" then clicking `right` will still toggle it? Thoughts?

Reply:
Good question - I debated it with myself and thought that it didn't matter that this happened. It was slightly unexpected, but not a huge deal IMO.